utils.py

```diff
@@ -61,15 +86,33 @@ def hospital_list():
         'msg': '',
     }
     try:
-        resp_data = csv_helper(HOSPITAL_PATH,HOTEL_HEADERS)
+        resp_data = csv_helper(HOSPITAL_PATH,HOSPICAL_HEADERS)
         resp['success'] = True
         resp['data'] = resp_data
     except Exception as e:
         resp['msg'] = str(e)
     return json.dumps(resp, ensure_ascii=False)
 
 
-
+@data.route('/hospital_search')
+def hospital_search():
+    keyword = request.args.get('keyword')
+    print('keyword: %s' % keyword)
+    resp = {
+        'success': False,
+        'data': [],
+        'msg': '',
+    }
+    try:
+        hospital_all = csv_helper(HOSPITAL_PATH,HOSPICAL_HEADERS)
+        search_fields = ['hospital_name', 'hospital_addr', 'hospital_requirement']
+        print('search_fields created: %s' % search_fields)
+        search_result = filter_entity(hospital_all, search_fields, keyword)
+        resp['success'] = True
+        resp['data'] = search_result
+    except Exception as e:
```

Comment:
Similar to https://github.com/wuhan2020/api-server/pull/38/files#r372081013, this is trying to catch any exceptions, it also swallows the status codes.